mbed_greentea/mbed_greentea_cli.py

```diff
@@ -782,7 +782,9 @@ def get_parallel_value(value):
                 if not mbed_dev['serial_port'].endswith(str(baudrate)):
                     mbed_dev['serial_port'] = "%s:%d" % (mbed_dev['serial_port'], baudrate)
                 mut = mbed_dev
-                muts_to_test.append(mbed_dev)
+                if mbed_dev not in muts_to_test:
```

Comment:
Wrote a quick bit of test code to check this and yes it seems to work fine. So LGTM